File: src/net/jimmc/swing/ToolPrompter.java
/* ToolPrompter.java
 *
 * Jim McBeath, June 1, 2002
 */

package net.jimmc.swing;

/** A component which can display tool prompt strings.
 */
public interface ToolPrompter {
	/** Display a tool prompt string. */
	public void showToolPrompt(String prompt);

	/** Clear the current tool prompt string. */
	public void clearToolPrompt();
}

/* end */

File: src/net/jimmc/swing/ToolPrompter.scala
/* ToolPrompter.scala
 *
 * Jim McBeath, June 1, 2002
 * converted from java to scala June 21, 2008
 */

package net.jimmc.swing

/** A component which can display tool prompt strings.
 */
trait ToolPrompter {
	/** Display a tool prompt string. */
	def showToolPrompt(prompt:String)

	/** Clear the current tool prompt string. */
	def clearToolPrompt()
}

/* end */
